xarray/tests/__init__.py

```diff
@@ -101,7 +101,7 @@
 try:
     _SKIP_FLAKY = not pytest.config.getoption("--run-flaky")
     _SKIP_NETWORK_TESTS = not pytest.config.getoption("--run-network-tests")
-except ValueError:
+except (ValueError, AttributeError) as e:
```

Comment:
did `AttributeError` come up for you?

Even if so, we don't need `as e` unless we're doing something with the error